Non-critical error when running Astro 5 in dev mode: "[vite] cannot find entry point module 'astro:db'."

### Astro Info

```block
Astro                    v5.0.0-beta.8
Node                     v20.10.0
System                   macOS (arm64)
Package Manager          npm
Output                   server
Adapter                  @astrojs/node
Integrations             @astrojs/svelte
                         @astrojs/tailwind
                         astro:db
                         @astrojs/db/file-url
```


### If this issue only occurs in one browser, which browser is a problem?

_No response_

### Describe the Bug

Running `npm run dev` shows:

```
18:33:24 [types] Generated 1ms

 astro  v5.0.0-beta.8 ready in 1114 ms

┃ Local    http://localhost:4321/
┃ Network  use --host to expose

▶ This is a  beta  prerelease build!
  Report issues here: https://astro.build/issues

18:33:24 watching for file changes...
18:33:24 [astro:db] New local database created.
18:33:24 [ERROR] [astro:db] [vite] cannot find entry point module 'astro:db'.
18:33:56 [astro:db] Seeded database.
....
```

Everything works though. 

### What's the expected result?

No error should be displayed.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-f2kqtd?file=src%2Fpages%2Findex.astro

### Participation

- [ ] I am willing to submit a pull request for this issue.

I can reproduce this error:
```
17:06:28 watching for file changes...
17:06:28 [astro:db] New local database created.
17:06:28 [ERROR] [astro:db] [vite] cannot find entry point module 'astro:db'.
17:06:32 [astro:db] Seeded database.
17:06:33 [200] / 1407ms
```

Also getting same output. v. 5.0.0 beta 7

```
19:08:20 [types] Generated 2ms
19:08:20 [vite] Re-optimizing dependencies because lockfile has changed
19:08:21 [astro:db] New local database created.
19:08:21 [ERROR] [astro:db] [vite] cannot find entry point module 'astro:db'.
```

I am facing the same error:

```text
13:35:56 [astro:db] New local database created.
13:35:56 [ERROR] [astro:db] [vite] cannot find entry point module 'astro:db'.
```

--- a/.changeset/red-emus-repair.md
+++ b/.changeset/red-emus-repair.md
@@ -1,0 +1,5 @@
+---
+'@astrojs/db': patch
+---
+
+Fixes a bug that caused an error to be logged about invalid entrypoints

--- a/packages/db/src/core/integration/index.ts
+++ b/packages/db/src/core/integration/index.ts
@@ -16,7 +16,7 @@ import {
 } from 'vite';
 import parseArgs from 'yargs-parser';
 import { AstroDbError } from '../../runtime/utils.js';
-import { CONFIG_FILE_NAMES, DB_PATH } from '../consts.js';
+import { CONFIG_FILE_NAMES, DB_PATH, VIRTUAL_MODULE_ID } from '../consts.js';
 import { EXEC_DEFAULT_EXPORT_ERROR, EXEC_ERROR } from '../errors.js';
 import { resolveDbConfig } from '../load-file.js';
 import { SEED_DEV_FILE_NAME } from '../queries.js';
@@ -153,7 +153,7 @@ function astroDBIntegration(): AstroIntegration {
 					);
 					// Eager load astro:db module on startup
 					if (seedFiles.get().length || localSeedPaths.find((path) => existsSync(path))) {
-						server.ssrLoadModule(resolved.module).catch((e) => {
+						server.ssrLoadModule(VIRTUAL_MODULE_ID).catch((e) => {
 							logger.error(e instanceof Error ? e.message : String(e));
 						});
 					}

